codalab/lib/codalab_manager.py

```diff
@@ -190,7 +190,7 @@ def init_config(self, dry_run=False):
                 'rest_host': 'localhost',
                 'rest_port': 2900,
                 'class': 'MySQLModel',
-                'engine_url': 'mysql://codalab@localhost:3306/codalab_bundles',
+                'engine_url': 'mysql://codalab@localhost:3306/codalab_bundles?charset=utf8',
```

Comment:
> I kept getting this issue (when running `worksheets` tests):
> 
> ```
> >> codalab/bin/cl wedit 0xf51f5551139042c78fa3534dc6e19187 --file /home/epicfaace/codalab-worksheets/tests/files/unicode-worksheet
> Traceback (most recent call last):
>   File "/home/epicfaace/codalab-worksheets/codalab/client/json_api_client.py", line 16, in wrapper
>     return f(*args, **kwargs)
>   File "/home/epicfaace/codalab-worksheets/codalab/client/json_api_client.py", line 680, in update_worksheet_raw
>     data='\n'.join(lines).encode(),
>   File "/home/epicfaace/codalab-worksheets/worker/codalabworker/rest_client.py", line 91, in _make_request
>     with closing(urllib.request.urlopen(request)) as response:
>   File "/usr/lib/python3.6/urllib/request.py", line 223, in urlopen
>     return opener.open(url, data, timeout)
>   File "/usr/lib/python3.6/urllib/request.py", line 532, in open
>     response = meth(req, response)
>   File "/usr/lib/python3.6/urllib/request.py", line 642, in http_response
>     'http', request, response, code, msg, hdrs)
>   File "/usr/lib/python3.6/urllib/request.py", line 570, in error
>     return self._call_chain(*args)
>   File "/usr/lib/python3.6/urllib/request.py", line 504, in _call_chain
>     result = func(*args)
>   File "/usr/lib/python3.6/urllib/request.py", line 650, in http_error_default
>     raise HTTPError(req.full_url, code, msg, hdrs, fp)
> urllib.error.HTTPError: HTTP Error 500: Internal Server Error
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "codalab/bin/../../codalab/bin/cl.py", line 125, in <module>
>     main()
>   File "codalab/bin/../../codalab/bin/cl.py", line 119, in main
>     cli.do_command(sys.argv[1:])
>   File "/home/epicfaace/codalab-worksheets/codalab/lib/bundle_cli.py", line 848, in do_command
>     structured_result = command_fn()
>   File "/home/epicfaace/codalab-worksheets/codalab/lib/bundle_cli.py", line 842, in <lambda>
>     command_fn = lambda: args.function(self, args)
>   File "/home/epicfaace/codalab-worksheets/codalab/lib/bundle_cli.py", line 2932, in do_wedit_command
>     client.update_worksheet_raw(worksheet_info['id'], lines)
>   File "/home/epicfaace/codalab-worksheets/codalab/client/json_api_client.py", line 38, in wrapper
>     sys.exc_info()[2],
>   File "/home/epicfaace/codalab-worksheets/venv/lib/python3.6/site-packages/six.py", line 692, in reraise
>     raise value.with_traceback(tb)
>   File "/home/epicfaace/codalab-worksheets/codalab/client/json_api_client.py", line 16, in wrapper
>     return f(*args, **kwargs)
>   File "/home/epicfaace/codalab-worksheets/codalab/client/json_api_client.py", line 680, in update_worksheet_raw
>     data='\n'.join(lines).encode(),
>   File "/home/epicfaace/codalab-worksheets/worker/codalabworker/rest_client.py", line 91, in _make_request
>     with closing(urllib.request.urlopen(request)) as response:
>   File "/usr/lib/python3.6/urllib/request.py", line 223, in urlopen
>     return opener.open(url, data, timeout)
>   File "/usr/lib/python3.6/urllib/request.py", line 532, in open
>     response = meth(req, response)
>   File "/usr/lib/python3.6/urllib/request.py", line 642, in http_response
>     'http', request, response, code, msg, hdrs)
>   File "/usr/lib/python3.6/urllib/request.py", line 570, in error
>     return self._call_chain(*args)
>   File "/usr/lib/python3.6/urllib/request.py", line 504, in _call_chain
>     result = func(*args)
>   File "/usr/lib/python3.6/urllib/request.py", line 650, in http_error_default
>     raise HTTPError(req.full_url, code, msg, hdrs, fp)
> codalab.client.json_api_client.JsonApiException: Unable to update worksheet: Internal Server Error - Unexpected Internal Error ('latin-1' codec can't encode characters in position 7-19: ordinal not in range(256)). The administrators have been notified.
>  (exit code 1, expected 0)
> ```
> 
> It appears that [908ce5c](https://github.com/codalab/codalab-worksheets/commit/908ce5c224ba75105aef1ceea0e73613bf812bd3) fixed it, although I'm not sure whether this is the best solution and whether this will work in production -- will need to test.

@percyliang see this comment regarding the database urls.